packages/core/test/lib/commands/test.js

```diff
@@ -1,14 +1,16 @@
 const assert = require("chai").assert;
 const Box = require("@truffle/box");
-const TestHelpers = require("@truffle/core/lib/commands/test/helpers");
+const {
+  determineTestFilesToRun
+} = require("../../../lib/commands/test/determineTestFilesToRun");
 const Artifactor = require("@truffle/artifactor");
 const Resolver = require("@truffle/resolver");
 const MemoryStream = require("memorystream");
 const path = require("path");
 const fs = require("fs-extra");
 const glob = require("glob");
 const WorkflowCompile = require("@truffle/workflow-compile");
-const Test = require("@truffle/core/lib/test");
+const Test = require("../../../lib/testing/Test");
```

Comment:
This is kind of ugly but do we really want to require `@truffle/core`?

Reply:
not sure it matters?